src/extensions/functions/checkRequirement.ts

```diff
@@ -22,6 +23,21 @@ module.exports = async (
     flatten
   )(requirement)
 
+  const taskWithFix = (checker: SolidarityChecker, rule: SolidarityRule, context: SolidarityRunContext) => async () => {
+    if (!shouldRunFix) return checker(rule, context)
+    try {
+      const result = await checker(rule, context)
+      return result
+    } catch (error) {
+      if (rule.fix) {
+        await context.system.run(rule.fix)
```

Comment:
I'm a tiny bit leery of immediately running the fix script with no extra confirmation. Other CLIs use a yes/no confirmation on each task, or a --yes option to bypass and run all. 

I realize this slightly complexifies the code, but might be safer when running a potentially environment altering script.

Reply:
This is an async function, so you can just add:

```js
context.print.info("Fix found for this issue:")
context.print.info(rule.fix)
const runFix = await context.prompt.ask("Should I run this fix for you?")
```

Reply:
It's worth noting that you already have to opt in by either passing the `--fix` flag or running the `fix` command. If we still want this it's easy enough to add though.

Reply:
yeah.... I'm with @ryanlntn - If you added the flag, it should bypass asking.